llama_hub/sec_filings/README.md

```diff
@@ -13,78 +13,27 @@ python install -r requirements.txt
 The SEC Downloader expects 5 attributes
```

Comment:
No, it requires 4 attributes now, instead of 5. The previous implementation is breaking, but the current implementation is directly pulling from the official page, hence it is more reliable. Currently, the users can pull all the files for a given year, and the amount parameter earlier was really ambiguous.

Reply:
I have done those changes in the README file